src/Orleans/CodeGeneration/GrainInterfaceUtils.cs

```diff
@@ -126,6 +126,9 @@ public static bool IsStatelessWorker(MethodInfo methodInfo)
         
         public static int ComputeMethodId(MethodInfo methodInfo)
         {
+            var attr = methodInfo.GetCustomAttribute<MethodIdAttribute>(true);
+            if (attr != null) return attr.MethodId;
```

Comment:
Precisely, @benjaminpetit. We could add this as a Roslyn Analyzer, but I figured that since it will fail compilation when the user adds a duplicate, it's safe to leave this unaddressed.